CHANGELOG.md

```diff
@@ -5,11 +5,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.1.2](https://github.com/livebook-dev/kino_db/tree/v0.1.2) (2022-06-30)
+
+### Added
+
+- Color required inputs and block source code generation when they're empty ([#20](https://github.com/livebook-dev/kino_db/pull/20))
+- Support for AWS Athena ([#15](https://github.com/livebook-dev/kino_db/pull/15))
+- Support for Google BigQuery ([#7](https://github.com/livebook-dev/kino_db/pull/7))
+- Support for SQLite ([#2](https://github.com/livebook-dev/kino_db/pull/2))
+- Warning when there's no available connection ([#11](https://github.com/livebook-dev/kino_db/pull/11))
+
+### Changed
+
+- Google BigQuery connection source with new Goth version ([#18](https://github.com/livebook-dev/kino_db/pull/18))
+- Google BigQuery help box when application is hosted on Google Cloud ([#19](https://github.com/livebook-dev/kino_db/pull/19))
```

Comment:
Since we added BigQuery in this release, these aren't changes to the end user :D You can add the PR numbers to the BigQuery item above.

Reply:
Good to know, I never did a release CHANGELOG 😝